### PR TITLE
v0.10.4 docfixes tlake (#490)

### LINKED ISSUE #490: Fixes needed for 0.10.4 docs
http://layer0.ims.io/guides/walkthrough/intro/

This guide aims to take you through **three** increasingly-complex deployment examples using Layer0.
Should say "two".

Deployment 3 link at the bottom of this page needs to be removed.

http://layer0.ims.io/reference/consul/

Admonition block under Consul Agent Container Definition needs to contain the line under the header.

http://layer0.ims.io/reference/task_definition/

Admonition block in the Sample section needs to contain the paragraph under the header.

http://layer0.ims.io/guides/walkthrough/deployment-2/

Cleanup section needs to remove the link to Deployment 3

### PR DESCRIPTION
**Pull request by tlake**
**What does this pull request do?**
Addresses documentation issues as per #490. Additionally, this PR removes the smoketests from the CI pipeline, since they are failing and will require more work than is within the scope of this PR to address.

**How should this be tested?**
`cd layer0/docs-src/ && make serve`, then browser at `localhost:9090`

**Checklist**
- ~[ ] Unit tests~
- ~[ ] Smoke tests (if applicable)~
- ~[ ] System tests (if applicable)~
- [x] Documentation (if applicable)

closes #490
(reference the issue(s) this pull request closes)

### CODE REVIEW

File: docs-src/docs/reference/task_definition.md [hunk 1]
```diff
@@ -31,10 +31,10 @@ The following snippet contains the task definition for the [Guestbook](../guides
 * **Name** The name of the container
 
 !!! warning
-If you wish to update your task definition, the container names **must** remain the same.
-If any container names are changed or removed in an updated task definition,
-ECS will not know how the existing container(s) should be mapped over and you will not be able to deploy the updated task definition.
-If you encounter a scenario where you must change or remove a container's name in a task definition, we recommend re-creating the Layer0 Deploy and Service.
+    If you wish to update your task definition, the container names **must** remain the same.
```
Comment (nehayward) [inline docs-src/docs/reference/task_definition.md]: Are these spaces consistent with other parts of the docs? Do we need to update elsewhere?

Comment (tlake) [reply docs-src/docs/reference/task_definition.md]: For any of the `!!!warning`-and-similar blocks, any text that should be included in the [admonition](https://squidfunk.github.io/mkdocs-material/extensions/admonition/) needs to be indented by four spaces, so this spacing should be consistent with any other not-broken admonition blocks.

Comment (nehayward) [reply docs-src/docs/reference/task_definition.md]: Thanks! I'll look back through and see if there are any other broken ones.

Comment (tlake) [reply docs-src/docs/reference/task_definition.md]: :+1: Awesome, thanks. I'm going through and doing the same, so it'll be good to have another set of eyes.

File: docs-src/docs/reference/cli.md [hunk 2]
```diff
@@ -231,10 +231,11 @@ Add-JobTrigger -Name $jobname -Trigger (New-JobTrigger -AtStartup -RandomDelay 0
 
 
 !!! note "Windows Environments"
-        Windows containers are still in beta. 
-You can view the documented caveats with ECS [here](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_Windows.html#windows_caveats).
-When creating Windows environments in Layer0, the root volume sizes for instances are 200GiB to accommodate the large size of the containers.  
-It can take as long as 45 minutes for a new windows container to come online. 
+    Windows containers are still in beta.
```
Comment (diemonster) [inline docs-src/docs/reference/cli.md]: technically, windows support is no longer beta https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_Windows.html

File: docs-src/docs/guides/walkthrough/deployment-2.md [hunk 3]
```diff
@@ -265,12 +265,10 @@ Your data should still be there!
 
 ### Cleanup
 
-If you're finished with the example and don't want to continue with this walkthrough, you can instruct Layer0 to delete the environment and terminate the application.
+When you're finished with the example, instruct Layer0 to delete the environment and terminate the application:
```
Comment (diemonster) [inline docs-src/docs/guides/walkthrough/deployment-2.md]: [nit] "instruct" is a odd word to use here

File: docs/guides/walkthrough/deployment-2/index.html [hunk 4]
```diff
@@ -1105,9 +1105,8 @@ <h3 id="part-7-prove-it">Part 7: Prove It<a class="headerlink" href="#part-7-pro
 Your data should still be there!</p>
 <hr />
 <h3 id="cleanup">Cleanup<a class="headerlink" href="#cleanup" title="Permanent link">#</a></h3>
-<p>If you're finished with the example and don't want to continue with this walkthrough, you can instruct Layer0 to delete the environment and terminate the application.</p>
+<p>When you're finished with the example, instruct Layer0 to delete the environment and terminate the application:</p>
```
Comment (diemonster) [inline docs/guides/walkthrough/deployment-2/index.html]: [nit] same re: "instruct"

File: docs/reference/cli/index.html [hunk 5]
```diff
@@ -3382,12 +3382,14 @@ <h4 id="optional-arguments">Optional arguments<a class="headerlink" href="#optio
 
 
 <div class="admonition note">
-<p class="admonition-title">Windows Environments<p>Windows containers are still in beta. </p>
-</p>
+<p class="admonition-title">Windows Environments</p>
+<p>Windows containers are still in beta. </p>
```
Comment (diemonster) [inline docs/reference/cli/index.html]: Same comment re: Windows beta

File: docs-src/docs/guides/walkthrough/deployment-2.md (hunk 3 above)
Comment (tlake) [reply docs-src/docs/guides/walkthrough/deployment-2.md]: It seems to be used similarly above in some of the argument summary sections. Do those also seem out of place, or is it just this one?

File: docs-src/docs/reference/cli.md (hunk 2 above)
Comment (tlake) [reply docs-src/docs/reference/cli.md]: Word - I still feel like the rest of the information in the admonition is helpful for a user, though.

File: Makefile [hunk 6]
```diff
@@ -41,10 +41,7 @@ install-smoketest:
 	$(MAKE) -C api release
 	$(MAKE) -C runner release
 
-apply-smoketest:
-	$(MAKE) -C setup apply-smoketest
-
 destroy-smoketest:
```
Comment (zpatrick) [inline Makefile]: we should remove `destroy-smoketest` as well

File: Makefile [hunk 7]
```diff
@@ -41,10 +41,7 @@ install-smoketest:
...
 	$(MAKE) -C setup destroy-smoketest
 
-.PHONY: release unittest smoketest install-smoketest apply-smoketest destroy-smoketest systemtest benchmark
+.PHONY: release unittest smoketest install-smoketest destroy-smoketest systemtest benchmark
```
Comment (zpatrick) [inline Makefile]: should remove `install-smoketest`, `destroy-smoketest`, and `benchmark`. Need to add `stresstest`

File: Makefile (hunk 6 above)
Comment (tlake) [reply Makefile]: ✅

File: Makefile (hunk 7 above)
Comment (tlake) [reply Makefile]: ✅

**Pull request merged**